exercises/02/readme.md

```diff
@@ -215,10 +218,14 @@ With a single command, you've initialized a basic OData service project and with
 
 ## Questions
 
-1. Why is there an focus on "TTM" (time to metadata) - what advantages does that bring?
+1. Why is there an focus on "Contracts First" (As all you need to run a service is a service definition) - what advantages does that bring?
+<!--- UI teams can start to work right away --->
 
-1. What is the difference between the data model and the service definition? Why do we need both?
+2. What is the difference between the data model and the service definition? Why do we need both?
```

Comment:
Any reason for moving to explicit numbering here (i.e. 1., 2., 3. instead of 1., 1., 1.)?

Reply:
The comments broke the automatic enumeration :( Maybe we should but the solutions in one single comment below the list to avoid this